tests/org.eclipse.fordiac.ide.test.ui/src/org/eclipse/fordiac/ide/test/ui/Abstract4diacUITests.java

```diff
@@ -353,6 +373,51 @@ protected static SWTBotGefEditor goToCompositeInstanceViewer(final String fb) {
 		return editor;
 	}
 
+	/**
+	 * Creates a New Type with given name.
+	 *
+	 * @param parentName Name of the parent project.
+	 * @param typeName   Name of the new type.
+	 */
+	protected static void createFBType(final String parentName, final String typeName) {
```

Comment:
what if you add an additional String parameter here so that we can select the kind of FB type that we want to create? e.g. Adapter, Composite, Struct etc. Maybe even an enum that helps us manage these Strings?